CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing
+
+Thank you for your interest in improving InsightConnect! Please review our [Code of Conduct] before making contributions.
+
+There are multiple ways to contribute beyond writing code. These include:
+
+- [Submit bugs and feature requests] with detailed information about your issue or idea.
+- [Help fellow users with open issues] or [help fellow committers test recent pull requests].
+- [Report a security vulnerability in InsightConnect's plugins] to Rapid7.
+- Submit an updated or brand new plugin!  We are always eager for new
+  integrations or features. Don't know where to start? Check out the [developer documentation].
+
+Here is a short list of do's and don'ts to make sure *your* valuable contributions actually make
+it into production.  If you do not care to follow these rules, your contribution
+**will** be closed. Sorry!
+
+## Code Contributions
+
+- **Do** read the [developer documentation]
+- **Do** stick to the [Python PEP8] and [plugin style] guides.
+- **Do** follow the [50/72 rule] for Git commit messages.
+- **Do** license your code as MIT.
+- **Do** create a [topic branch] to work on. This helps ensure users are aware of commits on the branch being considered for merge, allows for a location for more commits to be offered without mingling with other contributor changes, and allows contributors to make progress while a PR is still being reviewed.
+
+### Pull Requests
+
+- **Do** write "WIP" on your PR and/or open a [draft PR] if submitting **working** yet unfinished code.
+- **Do** target your pull request to the **master branch**.
+- **Do** specify a descriptive title to make searching for your pull request easier e.g. "Okta: add Suspend User action".
+- **Do** include [console output], especially the JSON output for new features and bug fixes.
+- **Do** list [verification steps] so your tests are reproducible.
+- **Do** [reference associated issues] in your pull request description.
+- **Don't** leave your pull request description blank.
+- **Don't** abandon your pull request. Being responsive helps us land your code faster.
+
+#### New Features
+
+- **Do** install validator dependencies necessary to run `make validate` to find and fix any errors or warnings that come up.
+- **Don't** include more than one plugin per pull request.
+- **Do** include documentation showing sample run-throughs.
```

Comment:
Can you put all the `do's` above the `don'ts`?

Reply:
Reordered